react/react1/week1/componentstables/src/App.js

```diff
@@ -0,0 +1,27 @@
+import "./App.css";
+import Table from "./Components/Table";
+
+function App() {
+  const todos = [
+    {
+      Description: "Get out of bed",
+      DeadlineDate: "Wed Sep 13 2017",
+    },
+    {
+      Description: "Brush teeth",
+      DeadlineDate: "Thu Sep 14 2017",
+    },
+    {
+      Description: "Eat breakfast",
+      DeadlineDate: "Fri Sep 15 2017",
+    },
+  ];
+
+  return (
+    <div className="App">
+      <Table todoData={todos}></Table>
```

Comment:
I suggest you write the components in this way.
```suggestion
      <Table todoData={todos} />
```